wp-content/themes/core/components/accordion/js/accordion.js

```diff
@@ -79,19 +79,27 @@ const openAccordion = ( header, content ) => {
 	setOffset();
 
 	slide.down( content, content.id, options.speed );
+
 	_.delay( () => {
-		scrollTo( {
-			after_scroll: () => {
-				events.trigger( {
-					event: 'modern_tribe/accordion_animated',
-					native: false,
-				} );
-			},
-
-			offset: options.offset,
-			duration: 300,
-			$target: $( row ),
-		} );
+		if ( ! el.container[ 0 ].dataset.scrollto ) {
+			events.trigger( {
+				event: 'modern_tribe/accordion_animated',
+				native: false,
+			} );
+		} else {
+			scrollTo( {
+				after_scroll: () => {
+					events.trigger( {
+						event: 'modern_tribe/accordion_animated',
+						native: false,
+					} );
+				},
+
+				offset: options.offset,
+				duration: 300,
+				$target: $( row ),
+			} );
+		}
```

Comment:
A couple things here to note:

1. `el.container[ 0 ]` will always be the first instance of an accordion component on the page, even when there are multiple instances. You'll have to adjust this to "find" the specific accordion container for the `header` that was clicked rather than just assuming the `0` instance.
2. We try to avoid else conditions as often as possible because it requires extra thought processing to grep and more room for errors. I would suggest an alternative here would be to do a `return;` after the even trigger in the first condition. Then the scroll to (else condition) can just exist as stand alone statements without the else condition wrapping it.
3. it's always good to reduce code duplication into separate methods whenever possible. I'd recommend moving the event trigger to it's own method and calling it in each conditional context rather than repeating it. (Notice I also suggest updated the `closeAccordion()` method since it too triggers the same event.)

Here's an example of these changes. Be sure to test them though. I didn't run them them anywhere before creating it:
```

/**
 * @function triggerAfterScroll
 * @description Handles triggering the accordion animated event.
 */
const triggerAfterScroll = () => {
	events.trigger( {
		event: 'modern_tribe/accordion_animated',
		native: false,
	} );
};

/**
 * @function openAccordion
 * @description Toggle the accordion open
 */

const openAccordion = ( header, content ) => {
	const component = tools.closest( header, '[data-js="c-accordion"]' );
	const row = tools.closest( header, '.c-accordion__row' );
	closeOthers( row );
	tools.addClass( row, 'active' );
	setAccActiveAttributes( header, content );
	content.removeAttribute( 'hidden' );
	setOffset();

	slide.down( content, content.id, options.speed );

	if ( ! component.dataset.scrollto ) {
		triggerAfterScroll();
		return;
	}

	_.delay( () => {
		scrollTo( {
			after_scroll: () => triggerAfterScroll,
			offset: options.offset,
			duration: 300,
			$target: $( row ),
		} );
	}, options.speed );
};

/**
 * @function closeAccordion
 * @description Toggle the accordion closed
 */

const closeAccordion = ( header, content ) => {
	const row = tools.closest( header, '.c-accordion__row' );
	tools.removeClass( row, 'active' );
	setAccInactiveAttributes( header, content );
	slide.up( content, content.id, options.speed );
	_.delay( () => {
		content.setAttribute( 'hidden', 'true' );
		triggerAfterScroll();
	}, options.speed );
};
```